elasticdl/python/master/k8s_instance_manager.py

```diff
@@ -359,30 +369,34 @@ def _event_cb(self, event):
 
     def get_alive_workers(self):
         alive_workers = []
-        for pod_name, phase in self._worker_pods_phase.values():
+        for pod_name, _, phase in self._worker_pods_ip_phase.values():
```

Comment:
We can get pod IP by 
```python
pod = k8s_client.get_pod(pod_name)
ip = pod.status.pod_ip
```
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1PodStatus.md

Reply:
After we record pod_ip, we don't need to call k8s API again.